index.html

```diff
@@ -12,54 +12,71 @@
   <div class="landing-page">
     <header>
       <nav class="nav">
-        <div class="nav__logo">my logo</div>
-        <div class="nav__menu-box">
-          <div class="nav__menu nav__menu--line-1"></div>
-          <div class="nav__menu nav__menu--line-2"></div>
-          <div class="nav__menu nav__menu--line-3"></div>
+        <div class="nav__logo"><a href="#" rel="noopener noreferrer">my logo</a></div>
+        <div class="nav__menu-line">
+          <div class="nav__menu"></div>
+          <div class="nav__menu"></div>
+          <div class="nav__menu"></div>
         </div>
+
+        <ul class="nav__menu-box">
+          <li class="nav__menu-item">
+            <a href="#portfolio-id">Portfolio</a>
```

Comment:
[OPTIONAL] This is only optional because it was not mentioned by the previous reviewer. However, It is part of the project requirements to make sure your design is as close to the Figma template as possible.  I highly recommend that you take it into consideration.

The right and left margin of the logo and menu items is way too large and does not match the margins on the Figma template.

Your Design:
![image](https://user-images.githubusercontent.com/38088818/165949686-d530a315-6a43-4b52-9171-723b07041847.png)


Figma:
![image](https://user-images.githubusercontent.com/38088818/165948732-037a4dc2-dc7f-4bb0-89b7-7c8857c3441d.png)